src/index.js

```diff
@@ -3,15 +3,17 @@ import ReactDOM from 'react-dom';
 import './index.scss';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { i18n } from '@lingui/core';
+import { I18nProvider } from '@lingui/react';
+//Start on eng
+import catalogEn from './locales/en/messages.js';
+i18n.load('en', catalogEn.messages);
```

Comment:
we should make it dynamic based on useres language from localstorage or navigator.language